src/Diagnostics.AIProjects/SearchAPI/SearchModule/resourceConfig.json

```diff
@@ -2,16 +2,20 @@
   "resourceConfig": {
     "App": {
       "WebApp": {
-        "Windows": "14748"
+        "Windows": "14748",
+        "Linux": "16170"
       },
       "ApiApp": {
-        "Windows": "14748"
+        "Windows": "14748",
+        "Linux": "16170"
       },
       "GatewayApp": {
-        "Windows": "14748"
+        "Windows": "14748",
+        "Linux": "16170"
```

Comment:
Hey Ajay, i think there are two PesId dor linux. One for web app linux and other for containers

Reply:
Okay will put both.